app/assets/javascripts/header.js

```diff
@@ -94,4 +68,36 @@ function setHeight() {
   $('#header_space').css({"height": the_height});
 }
 
+function expandButton(event) {
+  hideSmallTopNav();
+  console.log(event.data);
+  if (event.data.value % 2 == 0) {
```

Comment:
I'm not too much of a JS person myself. With that said, in english, what is `event.data.value` supposed to mean? Is there a way we can introduce a variable or function to provide a more readable name here?